packages/mongo/mongo_driver.js

```diff
@@ -1106,6 +1106,26 @@ _.extend(SynchronousCursor.prototype, {
   }
 });
 
+SynchronousCursor.prototype[Symbol.iterator] = function () {
+  var self = this;
+
+  // Get back to the beginning.
+  self._rewind();
+
+  return {
+    next: function () {
```

Comment:
What about method syntax? `next() {...}`

Reply:
Because the rest of the code is in old style, I use also here old style. Should I update just this part?

Reply:
Just this part is fine.  No harm in incremental modernization, especially when this is new code.